package.json

```diff
@@ -25,12 +25,23 @@
     "mkdirp": "^0.5.1",
     "ncp": "^2.0.0",
     "object.pick": "^1.3.0",
-    "tmp-promise": "^1.0.5"
+    "tmp-promise": "^1.0.5",
+    "jest": "latest",
+    "rimraf": "^2.6.3",
+    "util": "^0.11.1"
```

Comment:
`util` is something that comes with node, so we don't have to install it. https://www.npmjs.com/package/util